clang-tools-extra/test/clang-tidy/performance-unnecessary-value-param.cpp

```diff
@@ -166,6 +201,20 @@ void negativeValueNonConstMethodIsCalled(ExpensiveToCopyType Obj) {
   Obj.nonConstMethod();
 }
 
+void NegativeNoConstRefSinceMoved(ExpensiveToCopyType arg) {
+  auto F = std::move(arg);
+}
+
+template<typename T>
+T* NegativeNoConstRefSinceTypeMoved(ExpensiveToCopyType t) {
```

Comment:
There's no point to these template tests without an instantiation.

Reply:
Not really. A lot of the false positives that clang-tidy corrected where in templated constructors that were not instantiated. For this function too, clang-tidy flags it as an issue without any instantiation here - 

https://godbolt.org/z/_7Zc9e